packages/adapter-knex/lib/adapter-knex.js

```diff
@@ -507,6 +511,13 @@ class QueryBuilder {
 
     this._addWheres(w => this._query.andWhere(w), listAdapter, where, baseTableAlias);
 
+    // TODO: Implement configurable search fields for lists
+    const searchField = listAdapter.fieldAdaptersByPath['name'];
+    if (search !== undefined && searchField && searchField.fieldName === 'Text') {
+      this._query.whereRaw('true');
```

Comment:
We don't need the `whereRaw('true')` here, as we're now in the phase of the query builder where we're chaining together `andWhere` calls.